context/src/main/java/io/opentelemetry/context/StrictContextStorage.java

```diff
@@ -216,10 +216,10 @@ public String toString() {
     }
   }
 
+  // Don't care about serialization of this private class.
+  @SuppressWarnings("serial")
```

Comment:
Yeah, I think our intention is to not implement `Serializable` in a public API. I believe we don't right now, and this would be a reasonable reminder of that I guess